packages/css/tailwind.config.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+  prefix: 'vtmn-',
```

Comment:
@amen-souissi @gabbloquet @thollander @jleveugle what's your opinion about this prefix, please?

IMHO,
**Pros:**
- It ensures that every project that already uses Tailwind will not have conflicts.

**Cons:**
- It will make HTML & CSS more verbose (not good for performance in a big app)
- We will always have to tell "don't forget to add our prefix to utility classes, example: `vtmn-text-center` instead of `text-center`" when we will redirect the developer to Tailwind CSS documentation inside our Storybook.

Personally, now I think it will be better without prefix.

Reply:
Hey, I think that to facilitate adoption, we should prefix because it is a safe bet that Vitamin will not be the only one present in projects in transition.

For me, the argument about performances here is negligible, because I think that we have a lot of optimisation before that, like using only the design system (and remove others CSS), purge our CSS / JS etc.

I think that we can manage this purpose layer. Are you agreed ? cc @thollander

Reply:
@jleveugle thanks for your answer.
And what about the other problem in our documentation? Developers that use Vitamin will have to "take" the good documentation from Tailwind CSS without forget to add `vtmn-` before each utility classes.

Reply:
i agree with @jleveugle , it is possible that Vitamin is not the only design system used. to avoid any conflict, we should leave the prefix

Reply:
@jleveugle I agree with you. It is also for give an identity.
Another problem is that Vscode extension does not generate suggestions for Tailwind prefixes (hope this will be resolved soon ...)

Reply:
Thanks for your opinion! So we will keep this prefix.